draft-ietf-quic-multipath.md

```diff
@@ -1424,26 +1406,39 @@ MAX_PATHS frames are formatted as shown in {{fig-max-paths-frame-format}}.
 ~~~
 MAX_PATHS Frame {
   Type (i) = 0x15228c0b,
-  Maximum Paths (i),
+  Maximum Path Identifier (i),
 }
 ~~~
 {: #fig-max-paths-frame-format title="MAX_PATHS Frame Format"}
 
 MAX_PATHS frames contain the following field:
 
 Maximum Path Identifier:
-: A count of the cumulative number of path that can be opened
-  over the lifetime of the connection. This value cannot exceed 2^32-1, as it is not
-  possible to encode Path IDs larger than 2^32-1. Receipt of a frame that permits
-  opening of a path with Path ID larger than this limit MUST be treated
-  as a connection error of type FRAME_ENCODING_ERROR.
+: The latest Path ID that can be used for this connection.
+  This value cannot exceed 2^32-1, as it is not
+  possible to encode Path IDs larger than 2^32-1.
+
+Receipt of PATH_AVAILABLE, PATH_STANDBY, PATH_ABANDON or MP_ACK frames
+that uses a Path ID that is larger than the announced Path ID
+MUST be treated as a connection error of type FRAME_ENCODING_ERROR.
+
 
 Loss or reordering can cause an endpoint to receive a MAX_PATHS frame with
-a lower path limit than was previously received. MAX_PATHS frames that
-do not increase the path limit MUST be ignored.
+a smaller Path ID than was previously received. MAX_PATHS frames that
+do not announce a larger Path ID than previously received MUST be ignored.
+
+Endpoints SHOULD NOT issue new connection IDs which have path identifiers larger than
+the Path ID announced in the Maximum Path Identifier field in the
+MP_MAX_PATHS frame {{max-paths-frame}}.
+If no MAX_PATHS frame was received yet, the maximum Path ID
+correspondes to value of initial_max_paths transport parameter.
+
+An endpoint MUST NOT initiate a path with a Path ID larger than the Maximum Path Identifier value.
+An endpoint MUST terminate the a connection with an error of type MP_PROTOCOL_VIOLATION
+if a peer opens a path with a larger Path ID than permitted.
```

Comment:
To me, this whole paragraph can be removed once the corrections of the previous paragraph are integrated.

Reply:
+1 to remove it

Reply:
```suggestion
```